workos/async_client.py

```diff
@@ -1,72 +1,90 @@
 from workos._base_client import BaseClient
+from workos.audit_logs import AuditLogsModule
 from workos.directory_sync import AsyncDirectorySync
 from workos.events import AsyncEvents
+from workos.mfa import MFAModule
+from workos.organizations import OrganizationsModule
+from workos.passwordless import PasswordlessModule
+from workos.portal import PortalModule
 from workos.sso import AsyncSSO
+from workos.user_management import UserManagementModule
 from workos.utils.http_client import AsyncHTTPClient
+from workos.webhooks import WebhooksModule
 
 
 class AsyncClient(BaseClient):
     """Client for a convenient way to access the WorkOS feature set."""
 
     _http_client: AsyncHTTPClient
 
+    _audit_logs: AuditLogsModule
+    _directory_sync: AsyncDirectorySync
+    _events: AsyncEvents
+    _mfa: MFAModule
+    _organizations: OrganizationsModule
+    _passwordless: PasswordlessModule
+    _portal: PortalModule
+    _sso: AsyncSSO
+    _user_management: UserManagementModule
+    _webhooks: WebhooksModule
+
     def __init__(self, base_url: str, version: str, timeout: int):
         self._http_client = AsyncHTTPClient(
             base_url=base_url, version=version, timeout=timeout
         )
 
     @property
-    def sso(self):
+    def sso(self) -> AsyncSSO:
         if not getattr(self, "_sso", None):
             self._sso = AsyncSSO(self._http_client)
         return self._sso
 
     @property
-    def audit_logs(self):
+    def audit_logs(self) -> AuditLogsModule:
```

Comment:
This is temporarily while we don't have async support for these modules, but I'll work on adding.